mmdeploy/utils/test.py

```diff
@@ -511,11 +548,15 @@ def get_rewrite_outputs(wrapped_model: nn.Module,
             cfg=deploy_cfg, backend=backend.value, opset=11), torch.no_grad():
         ctx_outputs = wrapped_model(**model_inputs)
 
-    onnx_file_path = get_onnx_model(wrapped_model, model_inputs, deploy_cfg)
+    ir_type = get_ir_config(deploy_cfg).get('type', None)
+    if ir_type == 'torchscript':
```

Comment:
should we create and use `IR` enumerate data type?

Reply:
This part has been done in PR #150

Reply:
should we create and use IR enumerate data type since PR #150 is merged